### PR TITLE
node fetch

### PR DESCRIPTION
**Pull request by AbdullahAli**
Update `node-fetch` to latest: https://www.npmjs.com/package/node-fetch

Do not merge until we finish testing with `Sparkpost` to fix the issue with timeouts

### CODE REVIEW

File: app/services/sendService.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ function* send(userData, sendData, productInfo, espCampaignId) {
 		const response = yield fetch(config.espHost + path, options);
 
 		if (response.status >= 300) {
-			const errMsg = yield response.text();
+			const errMsg = yield response.textConverted();
```
Comment (AbdullahAli) [inline app/services/sendService.js]: https://github.com/bitinn/node-fetch/blob/master/UPGRADE-GUIDE.md#upgrade-to-node-fetch-v2x

**Comment by keirog** [comment]
Monday Merger™

**Comment by keirog** [comment]
Oh scrap that, it’s budget day on Monday! Tuesday Merger 😔

**Comment by AbdullahAli** [comment]
Closing for now to avoid bugs.  Worth a revisit when we introduce the new endpoint

**Pull request closed without merging**